### PR TITLE
Crud

### PR DESCRIPTION
**Pull request by VictorOfCode**
Please review

### CODE REVIEW

File: server/server.js [hunk 1]
```diff
@@ -13,23 +14,63 @@ app.use(express.static(path.resolve(__dirname, '../client/dist')));
...
     if (err) {
       console.log(err);
     }
   })
 })
 
-app.get('/projects/:projectId/comments', cors(), (req, res) => {
-  Project.find({"projectId": req.params.projectId}, (err, results) => {
+app.get('/projects/:projectId/comments', (req, res) => {
+  Comment.find({"projectId": req.params.projectId}, (err, results) => {
```
Comment (jcelder) [inline server/server.js]: @vyoung415 Is there another way we could refer to this object?

File: server/server.js [hunk 2]
```diff
@@ -13,23 +14,63 @@ app.use(express.static(path.resolve(__dirname, '../client/dist')));
...
+    } else {
+      res.send('posted');
+    }
+  });
+})
+
+// update 
+app.put('/projects/:projectId/comments/:commentId', (req, res) => {
+  Comment.findOneAndUpdate({
+    projectId: req.params.projectId,
```
Comment (jcelder) [inline server/server.js]: @vyoung415 Is there another way we could refer to this object?

File: server/server.js [hunk 3]
```diff
@@ -13,23 +14,63 @@ app.use(express.static(path.resolve(__dirname, '../client/dist')));
...
+      res.send(err);
+    }
+    res.send('updated');
+  });
+});
+
+// delete single comment
+app.delete('/projects/:projectId/comments/:commentId', (req, res) => {
+  Comment.findOneAndDelete({
+    projectId: req.params.projectId,
```
Comment (jcelder) [inline server/server.js]: @vyoung415 Is there another way we could refer to this object?

File: README.md [hunk 4]
```diff
@@ -1,11 +1,38 @@
...
+STEP 1: Dependencies
+npm install
+npm run start
+npm run dev
 
-This is a single module in a 4 module application. The project's repo can be found at: https://github.com/FEC-Kickstand
-My repository for this single comments module can be found at: https://github.com/FEC-Kickstand/comments-module
+STEP 2: Database
+mongod
+mondgo
```
Comment (jcelder) [inline README.md]: @vyoung415 Is this a spelling error?

**Review by jcelder — CHANGES_REQUESTED** [review]
@vyoung415 It looks good, though there are a few changes I would make. Let me know when you've made them and I will re-review this PR.

File: README.md (hunk 4 above)
Comment (VictorOfCode) [reply README.md]: Good catch, resolved error.

**Review by jcelder — APPROVED** [review]

**Pull request merged**